### ISSUE TITLE
JSX causes the module not to properly load depending on your configuration

### ISSUE THREAD

**Issue by BDiehr** [Moosylvania/react-growl#2]
Depending on how someone has their configurations set up, someone might not have a jsx-harmony loader parsing the node_modules folder. I think it's worth providing a vanilla javascript version as the production npm package.

**Comment by nuclearspike** [comment]
I agree. This is the worst install experience of a react component I've ever had. I should be able to just require/import from 'react-growl' and that's it.
